js_modules/dagit/packages/core/src/runs/RunTimeline.tsx

```diff
@@ -294,29 +357,46 @@ const TimeDividers = (props: TimeDividersProps) => {
       .map((_, ii) => {
         const time = firstMarker + ii * interval;
         const date = new Date(time);
-        const label = date.toLocaleString(locale, {
-          hour: '2-digit',
-          timeZone: timezone === 'Automatic' ? browserTimezone() : timezone,
```

Comment:
I'm not sure how slow this call is but it might make sense to pull it out to a variable outside the `.map`, just in case it's not speedy